feather/common/src/events.rs

```diff
@@ -8,7 +8,10 @@ use crate::view::View;
 mod block_change;
 mod plugin_message;
 
+pub mod health;
+
 pub use block_change::BlockChangeEvent;
+pub use health::HealthEventType;
 pub use plugin_message::PluginMessageEvent;
 
 /// Triggered when a player joins the `Game`.
```

Comment:
Might be better to just use game without the \` since its not a type, term, or variable name.

Reply:
The backticks around "Game"

Reply:
Oh, that's not my change.

Reply:
Welp, might as well fix it while we're here I guess.